services/openvpn/client.go

```diff
@@ -89,6 +90,13 @@ func (c *Client) GetConfig() (connection.ConsumerConfig, error) {
 		return nil, errors.Wrap(err, "failed to get consumer config")
 	}
 	c.publicIP = ip
+
+	switch c.natPinger.(type) {
+	case *traversal.NoopPinger:
+		log.Infof("noop pinger detected, returning nil client config.")
+		return nil, nil
+	}
```

Comment:
It looks like configuration depends on implementation of `traversal.Pinger` - maybe it should provide the config to avoid type-conditionals?

Reply:
Not sure if it is a bad solution, since introducing another config dependency will additionally span throughout the code. Does if worth it?